packages/carousel/src/Carousel.jsx

```diff
@@ -243,4 +244,6 @@ Carousel.propTypes = {
   displayArrowsMobile: PropTypes.bool,
   /** Set the height and width of the forward and back buttons to this value, e.g. "30px" */
   buttonSize: PropTypes.string,
+  /** When arrow position is side, hide arrows and shows when hovers on carousel */
+  showArrowsOnHover: PropTypes.bool,
```

Comment:
@davidjeonprice, is there a reason to restrict this to the `side` position?

Reply:
@craigpalermo yes, only `side` button appears on the top of a content.